pages/_app.tsx

```diff
@@ -10,15 +10,16 @@ import '@styles/globals.css';
 import { Header } from '../components/header/Header';
 import SideBar from '@components/SideBar';
 import SideNav from '@components/SideNav';
+import TableOfContent from '@components/TableOfContent';
 
 const Grid = styled.div`
     display: grid;
-    grid-template-columns: 0rem 100vw;
+    grid-template-columns: 0rem 100vw 0rem;
```

Comment:
```suggestion
    grid-template-columns: 0 100vw 0;
```

When you have `0` you don't need the unit.